core/vendor/bootstrap/js/npm.js

```diff
@@ -0,0 +1,13 @@
+// This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
```

Comment:
autogenerated file, why is tracked and no in gitignore? also, do we need new boostrap?